console/console-init/ui/src/modules/address-space/components/AddressSpaceToolbar/AddressSpaceToolbar.tsx

```diff
@@ -91,8 +97,8 @@ export const AddressSpaceToolbar: React.FunctionComponent<IAddressSpaceToolbarPr
       )}
       <DataToolbarItem>
         <AddressSpaceListKebab
-          createAddressSpaceOnClick={onCreateAddressSpace}
-          onDeleteAll={onDeleteAll}
+          onCreateAddressSpace={onCreateAddressSpace}
+          onKebabSelect={onSelectKebabDeleteAll}
```

Comment:
What was the reason behind changing the name 'onDeleteAll'? Does it check which kebab was selected inside the event handler, if not, then onDeleteAll makes more sense.

Reply:
We can't have onDeleteAll becasuse onDeleteAll function is already there which coming as props. so make it onSeletecDeleteAll.